codeguru_profiler_agent/sampling_utils.py

```diff
@@ -87,6 +87,12 @@ def _maybe_append_synthetic_frame(result, frame, line_no):
 
 def _extract_frames(end_frame, max_depth):
     stack = list(traceback.walk_stack(end_frame))[::-1][0:max_depth]
+    # When running the sample app with uwsgi for Python 3.8.10-3.9.2, the traceback command
+    # returns a file path that contains "/.".
+    # To not let the path go into the module name, we are removing it later in the ProfileEncoder.
+    # Examples:
+    # - file '/Users/mirelap/Documents/workspace/JSON/aws-codeguru-profiler-python-demo-application/sample-demo-django-app/./polls/views.py', line 104, code get_queryset>, 104
+    # - file '/Users/mirelap/Documents/workspace/JSON/aws-codeguru-profiler-python-demo-application/sample-demo-django-app/polls/views.py', line 104, code get_queryset>, 104
```

Comment:
this comment seems to be very specific about the /./ problem but is not next to the code that actually fixes this issue, should we not simply add a comment here saying only that the full path will be removed from frame names later in the encoder and only talk about the /./ issue when we do the `replace` about it?

Reply:
Okay, I'll update.